### DISCUSSION TITLE
ödev

### DISCUSSION THREAD

**Discussion by B3Y0N4** [19/classroom#23] [Anonim] [answered]
![hata](https://user-images.githubusercontent.com/72824981/158034100-34dddbda-fd0e-4ac0-b7ad-965866f12489.jpg)
Ödevim değlendiryeme almiyor hatada vemiyor lets grad komutuda algilamiyor ne yapmam gerekiyor

**Comment by roktas** [comment]
Windows üzerinde çalışıyorsun hala.  Dokümanları okudun mu?  https://classroom.alaturka.dev/guide/#vs-code-icinden-wsl-e-erisim

Dokümanları yeterince dikkate almıyorsunuz.  Çözüm bulamıyorsan ödevi tarayıcı üzerinde yap.

**Comment by ilkertoklu** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/92054649/158081665-0ab7d8d4-5664-4e6b-9d4b-83cf4d6e2d80.png)
Dosyayi WSL`de actiktan sonra Linux terminali uzerinde calisabilirsin.

**Comment by roktas** [comment]
@ilkertoklu teşekkürler, yardım çaban güzel ama...  Ben boşuna mı hazırladım o kadar animasyonu? (sözüm sana değil)
